### DISCUSSION TITLE
Welcome to FAST-Data-Science-Material Discussions!

### DISCUSSION THREAD

**Discussion by fatimaazfar** [fatimaazfar/FAST-Data-Science-Material#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gemyvic** [comment]
Hello everyone 
I'm Ahmed , I'm looking forward to studying data science and I inquire how to join the program

**Reply by fatimaazfar** [reply]
If you're looking to join FAST. You'll have to apply for the admissions which will probably open this month. You'll have to pass the entry test and the results combined with your previous results would make up your merit, if you're in the range, you'll get in. To prepare for the exam, you should go to FAST university's website to look at last papers and prepare well.

**Comment by samiaa7** [comment]
Hello, I will be taking the Advanced Statistics course next semester. Could you share a bit more resources like the course outline or the reference books it would really help. You have a great repo for FAST students, Thanks!
